resources/js/processes-catalogue/components/ProcessInfo.vue

```diff
@@ -28,13 +28,14 @@ import ProcessesCarousel from "./ProcessesCarousel.vue";
 import ProcessMap from "./ProcessMap.vue";
 import ProcessOptions from "./ProcessOptions.vue";
 import ProcessTab from "./ProcessTab.vue";
-
+import ChartSaveSearch from "./optionsMenu/ChartSaveSearch.vue";
```

Comment:
this is not necessary
It's defined in the ProcessOptions

Reply:
Solved